### ISSUE TITLE
File manager auto-updates

### ISSUE THREAD

**Issue by mattpass** [icecoder/ICEcoder#868]
The file manager can auto update after running indexer.php and finds new dirs/files.

We can then remove the refresh icon as it's self refreshing.

**Comment by cedricgeffroy** [comment]
Even if it auto refreshes I would personally prefer keep the refresh button. frustration-free UX ?

**Comment by mattpass** [comment]
Yeah, can do incase it's needed. My hope is it's not needed in future.
